extensions/interactions/base_test.py

```diff
@@ -34,7 +34,7 @@
 
 # File names ending in any of these suffixes will be ignored when checking the
 # validity of interaction definitions.
-IGNORED_FILE_SUFFIXES = ['.pyc', '.DS_Store']
+IGNORED_FILE_SUFFIXES = ['.pyc', '.DS_Store', '.html~']
```

Comment:
Hm. Files ending with ~ should just be deleted. (The others can arise in normal operation.)

There should be an option in gedit (or whatever text editor you're using) to disable this, though you could also use other text editors like Sublime Text.